src/test/scala/sigmastate/helpers/SigmaTestingCommons.scala

```diff
@@ -79,16 +88,19 @@ trait SigmaTestingCommons extends PropSpec
     val tB = RType[B]
     val tpeA = Evaluation.rtypeToSType(tA)
     val tpeB = Evaluation.rtypeToSType(tB)
+    val getVarType = if (tpeA == SBigInt) "BigInt" else tA.name
```

Comment:
this hack it not necessary, tA.name should return correct type name. See `implicit val BigIntRType: RType[BigInt] = wrapperType[BigInt]`

Reply:
I've switched to `special.sigma.BigInt` and after a couple of fixes [commit](
https://github.com/ScorexFoundation/sigmastate-interpreter/pull/400/commits/55eb2d198f02653908efd1485c65d9d93d370299) I ended up with a path type which drives our parser crazy:
```
line 3:   val res = func(getVar[special.sigma.BigInt](1).get)
                                              ^
Path types are not supported: Select(Select(Ident(special,NoType),sigma,None),BigInt,None)
```
For `scala.BigInt` case I stuck with missing ev for RType. I'll get back to that later.

Reply:
Yes, scala.BigInt is not and will not be supported. There is BigIntegerRType and BigIntRType implicits is special.sigma package object. Note, that BigInt is parsed string is not the same type as special.sigma.BigInt. They correspond, but not the same things.

Reply:
@aslesarenko Ok, I'm a bit confused here. Do we have to support both `special.sigma.BigInt` and `java.math.BigInteger` in SigmaDsl? If so, does it mean that both of them in the generated script should have `BigInt` sigma type in `getVar[BigInt](1)`? 
`tA.name` for both of them renders a path type which is not supported in our parser. Do you see another way of fixing instead of my original hack above? I don't like changing `SGeneralType.name` to not return a path type.

Reply:
In SigmaDsl we only use special.sigma.BigInt, which is WrapperOf[BigInteger] (see WrapperOf)
BigInterger we use internally.

You can add this method.
```
  case class WrapperType[Wrapper](cWrapper: ClassTag[Wrapper]) extends RType[Wrapper] {
   ...
    override def name: String = cWrapper.runtimeClass.getSimpleName
  }
```
Note, in `def func[A: RType, B: RType](func: String)(implicit IR: IRContext): A => B` we can assume A and B both SigmaDsl types.
See  this function for details of data conversion
```
/** Generic converter from types used in ErgoTree values to types used in ErgoDsl. */
  def toDslData(value: Any, tpe: SType, isCost: Boolean)(implicit IR: Evaluation): Any
```

Reply:
Separation between SigmaDsl types and ErgoTree types is annoying, but maybe in future we can eliminate this distinction. Currently we need to do conversions. Search for places where conversion methods are used.

Reply:
Done in 
https://github.com/ScorexFoundation/sigmastate-interpreter/pull/400/commits/2a5524a9b61e01af4bab4613002ef883057da40f